libinstall/functions

```diff
@@ -1271,6 +1287,85 @@ copy_ssh_keys_to_gorgone() {
     return 0
 }
 
+#----
+## Create gorgone configuration folders
+#----
+create_gorgone_configuration_structure() {
+    ## create the required gorgone configuration structure
+    conf_folder="$GORGONE_CONFIG/config.d"
+    if [ ! -d "$conf_folder" ] ; then
+        mkdir -p "$conf_folder"
+        if [ $? -ne 0 ] ; then
+            echo_failure "$(gettext "Cannot create the subfolder") $conf_folder" "$fail"
+            return 1
+        fi
+    fi
+    log "INFO" "GORGONE_CONFIG/config.d: $conf_folder"
+
+    # gorgone's config.yaml
+    destination="$GORGONE_CONFIG/config.yaml"
```

Comment:
Gorgone's config.yaml should already be installed by Gorgone installer. Why do you care about it here ? IMHO lines 1305 to 1320 should be removed.

Reply:
You're right. I'll check again